pkg/orchestrator/assetscanprocessor/processor.go

```diff
@@ -68,49 +63,49 @@ func (asp *AssetScanProcessor) Reconcile(ctx context.Context, event AssetScanRec
 	}
 
 	// Process each of the successfully scanned (state DONE and no errors) families into findings.
-	if statusCompletedWithNoErrors(assetScan.Status.Vulnerabilities) {
+	if assetScan.Vulnerabilities.Status.State == models.Done {
```

Comment:
I think we need to check for `nil` like we were doing inside `statusCompletedWithNoErrors()`, right?